src/transformers/commands/lfs.py

```
@@ -0,0 +1,219 @@
+
+import requests
+from transformers.commands import BaseTransformersCLICommand
+
+from ..utils import logging
+
+
+logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
+
+
+LFS_MULTIPART_UPLOAD_COMMAND = "lfs-multipart-upload"
+
+
+class LfsCommands(BaseTransformersCLICommand):
+    # Implementation of a custom transfer agent for the transfer type "multipart" for git-lfs.
+    # This lets users upload large files >5GB 🔥.
+    # Spec for LFS custom transfer agent is: https://github.com/git-lfs/git-lfs/blob/master/docs/custom-transfers.md
+    #
+    # This introduces two commands to the CLI:
+    #
+    #     transformers-cli lfs-enable-largefiles
+    #
+    # Do this once per model repo where you want to push >5GB files.
+    # It's documented in the error message you get if you just try to git push a 5GB file
+    # without having enabled it before.
+    #
+    #     transformers-cli lfs-multipart-upload
+    #
+    # is the custom transfer agent itself. This is not meant to be
+    # called by the user, but by lfs directly.
+
+    @staticmethod
+    def register_subcommand(parser: ArgumentParser):
+        enable_parser = parser.add_parser(
+            "lfs-enable-largefiles", help="Configure your repository to enable upload of files > 5GB"
+        )
+        enable_parser.add_argument("path", type=str, help="Local path to repository you want to configure.")
+        enable_parser.set_defaults(func=lambda args: LfsEnableCommand(args))
+
+        upload_parser = parser.add_parser(
+            LFS_MULTIPART_UPLOAD_COMMAND, help="This will get called by git-lfs, do not call it directly."
+        )
+        upload_parser.set_defaults(func=lambda args: LfsUploadCommand(args))
+
+
+class LfsEnableCommand:
+    def __init__(self, args):
+        self.args = args
+
+    def run(self):
+        local_path = os.path.abspath(self.args.path)
+        if not os.path.isdir(local_path):
+            print("This does not look like a valid git repo")
```

maybe better to raise an error?